### PR TITLE
@mzikherman => [Bugfix] Pagination in sale_artworks

### PR DESCRIPTION
**Pull request by damassi**
Fixes an issue where pagination wasn't being properly passed to relay.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ function startApp(schema) {
   app.use(bodyParser.json())
 
   if (enableQueryTracing) {
-    console.warn("[FEATURE] Enabling query tracing")
+    console.warn("[FEATURE] Enabling query tracing") // eslint-disable-line
```
Comment (mzikherman) [inline index.js]: yesss thank you

File: schema/sale_artworks.js [hunk 2]
```diff
@@ -7,9 +7,7 @@ import { SaleArtworkAggregations, SaleArtworkCounts } from "schema/filter_sale_a
 
 const DEFAULTS = {
   aggregations: ["total"],
-  page: 1,
-  size: 10,
-  offset: 0,
+  first: 10,
```
Comment (mzikherman) [inline schema/sale_artworks.js]: Cool. Nice work on the default for this, I never remember `first` and always spend some minutes wondering why the connection returned an empty list.

File: schema/sale_artworks.js [hunk 3]
```diff
@@ -26,11 +24,12 @@ export default {
   description: "Sale Artworks Elastic Search results",
   type: SaleArtworksType,
   resolve: async (_root, options, _request, { rootValue: { saleArtworksFilterLoader } }) => {
-    const params = parseRelayOptions({ ...DEFAULTS, ...options })
+    const relayOptions = { ...DEFAULTS, ...options }
```
Comment (mzikherman) [inline schema/sale_artworks.js]: good renames

File: schema/sale_artworks.js (hunk 2 above)
Comment (orta) [reply schema/sale_artworks.js]: good call, maybe we should do that in all connections?

**Pull request merged**